tools/code-owners-parser/CodeOwnersManualTester/CodeOwnerUtils.cs

```diff
@@ -0,0 +1,24 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Azure.Sdk.Tools.CodeOwnersParser;
+
+namespace CodeOwnersManualTester
+{
+    internal class CodeOwnerUtils
+    {
+        /// <summary>
+        /// Wrapper function to load the CODEOWNERS file from a given path or URL and return
+        /// the list of codeowners entries.
+        /// </summary>
+        /// <param name="codeOwnersFilePath"></param>
+        /// <returns>List of CodeownersEntry</returns>
+        public static List<CodeownersEntry> GetCodeOwnerEntries(string codeOwnersFilePath)
```

Comment:
I feel this method belongs in the `CodeownersFile` class.

Reply:
The wrapper class is gone. Will be updated with the next commit.